src/controllers/message.js

```diff
@@ -1,52 +1,42 @@
+const errors = require("../utils/error/errors");
 module.exports = [
   "msgRepository",
   (msgRepository) => {
     return {
       newMessage: async (req, res, next) => {
-        const time = new Date(req.body.time);
-
-        const query = {
-          text: req.body.text,
-          author: req.body.author,
-          room: req.body.room,
-          isPinned: req.body.isPinned,
-          time: time,
-        };
-
-        const opt = {
-          query,
-        };
-
-        const result = await msgRepository.create(opt);
-
-        res.json(result);
-      },
-
-      insertMany: async (reqs, res, next) => {
-        const queries = [];
+        try {
+          if (req.body.text === "") {
```

Comment:
Thinks you can just do !req.body.text for checking if its an empty string or null

Reply:
!req.body.text will be true if req.body does not contain a field called text.

Reply:
Yes. Shouldn't the field exist and not be empty? If its not then there isn't any point in creating it anyways. It would just throw a mongo error. Unless you want to handle it at that point.

Reply:
yeah this check is basically only checking if the field is empty, i should also check if it exists.

Reply:
Okay sure that's fine then.